packages/colorpicker/lib/pages/pipette_tool_page.dart

```diff
@@ -0,0 +1,144 @@
+import 'dart:async';
+import 'dart:typed_data';
+import 'dart:ui' as ui;
+
+import 'package:colorpicker/src/components/top_bar.dart';
+import 'package:colorpicker/src/state/color_picker_state_provider.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class PipetteToolPage extends ConsumerStatefulWidget {
```

Comment:
Opening a new page for pipette tool is overkill. As mentioned in my previous comment the kotlin version of Pocket Paint doesnt open a new page. You can approach this tool the easiet way:

1. onTap -> get color of the pixel under the point
2. set paint color to that color using paint provider

This simplyfies the user experience and matches the original implementation of the tool and removes alot of unnessary code.